test_haystack/solr_tests/test_solr_backend.py

```diff
@@ -1458,7 +1458,7 @@ def test_boost(self):
         ])
 
 
-@unittest.skipIf(pysolr.__version__ < (3, 1, 1), 'content extraction requires pysolr > 3.1.0')
+@unittest.skipIf(tuple(pysolr.__version__.split('.')) < (3, 1, 1), 'content extraction requires pysolr > 3.1.0')
```

Comment:
This could also use `pysolr.version_info` but that will trigger a deprecation warning on setuptools >= 8.0 since that's no longer a simple tuple.

Reply:
That actually seems to have an additional problem as the parsed version still contains strings instead of integers. 
Either we could do a `pysolr.version_info < parse_version('3.1.1')` with the corresponding import or we just drop that annotation? Seems relatively unlikely that anyone is running the test suite with a version of pysolr from 2013, no?
What do you prefer?

Reply:
Dropping it does make sense - if anyone is running tests on an install that old it seems unlikely that they'd have upgraded haystack but frozen pysolr

Reply:
Okay, I dropped the annotation.